AutoPartsShop.API/Controllers/ShoppingCartsController.cs

```diff
@@ -0,0 +1,168 @@
+﻿using AutoPartsShop.API.Extensions;
+using AutoPartsShop.API.Repositories.Interfaces;
+using AutoPartsShop.Models.DTOs;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AutoPartsShop.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ShoppingCartsController : ControllerBase
+    {
+        private readonly IShoppingCartRepository shoppingCartRepository;
+        private readonly IProductRepository productRepository;
+
+        public ShoppingCartsController(IShoppingCartRepository shoppingCartRepository,
+                                      IProductRepository productRepository)
+        {
+            this.shoppingCartRepository = shoppingCartRepository;
+            this.productRepository = productRepository;
+        }
+
+        [HttpGet]
+        [Route("{userId}/GetItems")]
+        public async Task<ActionResult<IEnumerable<CartItemDTO>>> GetItems(int userId)
```

Comment:
Same comment about the names is applicable here as well.
`GetItems` should become `GetShoppingCarts`, `GetItem` should become `GetShoppingCart` and so on